packages/teleport/src/components/Empty/Empty.tsx

```diff
@@ -0,0 +1,171 @@
+            {props.clusterId}
+          </Text>
+          " cluster.
+        </Text>
+      </Box>
+    );
+  }
+
+  return (
+    <Card
+      as={Flex}
+      width="879px"
+      height="232px"
+      mt={4}
+      mx="auto"
+      bg="primary.main"
+      alignItems="center"
+    >
+      <QuickstartContainer onClick={() => window.open(videoLink)}>
+        <Image width="265px" src={graphic} />
+        <Box style={{ position: 'absolute' }} mt={4}>
+          <Image m="auto" mb={1} width="90px" src={playIcon} />
+          <Text fontWeight={700}>WATCH THE QUICKSTART</Text>
+        </Box>
+      </QuickstartContainer>
+      <Box m="auto" mt={4}>
+        <Box width="500px" mb={4}>
+          <Text typography="h3" mb={2} fontWeight={700} fontSize={14}>
+            {title}
+          </Text>
+          <Text>{description}</Text>
+        </Box>
+        <ButtonPrimary onClick={onButtonClick} width="224px" height="40px">
+          {buttonText}
+        </ButtonPrimary>
+      </Box>
+    </Card>
+  );
+}
+
+const emptyStateInfo = {
+  nodes: {
+    title: 'ADD YOUR FIRST SERVER',
+    description:
+      'Centralized control and access to Servers with Teleport Server Access. Add labels to nodes and get full visibility into access and behavior.',
+    buttonText: 'ADD SERVER',
+    videoLink: 'https://www.youtube.com/watch?v=tUXYtwP-Kvw',
+    graphic: nodesEmptyPng,
+    readOnly: {
+      title: 'No Servers Found',
+      message: 'There are no servers for the "',
+    },
+  },
+  applications: {
+    title: 'ADD YOUR FIRST APPLICATION',
+    description:
+      'Teleport Application Access provides secure access to internal applications without the need for a VPN but with the audibility and control of Teleport.',
+    videoLink: 'https://www.youtube.com/watch?v=HkBQY-uWIbU',
+    buttonText: 'ADD APPLICATION',
+    graphic: emptyPng,
+    readOnly: {
+      title: 'No Applications Found',
+      message: 'There are no applications for the "',
+    },
+  },
+  kubernetes: {
+    title: 'ADD YOUR FIRST KUBERNETES CLUSTER',
+    description:
+      'Teleport Application Access provides secure access to internal applications without the need for a VPN but with the audibility and control of Teleport.',
+    videoLink: 'https://www.youtube.com/watch?v=2diX_UAmJ1c',
+    buttonText: 'VIEW DOCUMENTATION',
+    graphic: emptyPng,
+    readOnly: {
+      title: 'No Kubernetes Clusters Found',
+      message: 'There are no kubernetes clusters for the "',
+    },
+  },
+  databases: {
+    title: 'ADD YOUR FIRST DATABASE',
+    description:
+      'Teleport Application Access provides secure access to internal applications without the need for a VPN but with the audibility and control of Teleport.',
+    videoLink: 'https://www.youtube.com/watch?v=PCYyTecSzCY',
+    buttonText: 'ADD DATABASE',
+    graphic: emptyPng,
+    readOnly: {
+      title: 'No Databases Found',
+      message: 'There are no databases for the "',
+    },
+  },
+};
```

Comment:
Instead of having 1 place that describes these parameters for all possible scenarios, let each use-case to explicitly set and define own params. This not only makes it easier to find/locate a necessary change but also let deleting the code easier without any leftovers. It becomes even more useful when code that uses it resides in different repos (oss/e)